server/src/js/components/map/layers.js

```diff
@@ -0,0 +1,230 @@
+'use strict'
```

Comment:
I appreciate the commented out code is to store the different layers we need to investigate, but may be worth putting on a feature branch.